Source/BoundaryConditions/WarpX_PEC.H

```diff
@@ -45,11 +45,13 @@ namespace PEC {
      * \param[in,out] Bfield              Boundary values of normal Bfield are set to zero.
      * \param[in]     field_boundary_lo   Boundary types of the "low" field boundaries
      * \param[in]     field_boundary_hi   Boundary types of the "high" field boundaries
+     * \param[in]     bc_type             TODO
```

Comment:
Copy the suggestion in https://github.com/BLAST-WarpX/warpx/pull/3560/files?diff=unified&w=1#r2232272152.

Reply:
```suggestion
     * \param[in]     bc_type             Boundary condition type to be applied
```